spec/features/user_adds_review_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+feature "users can add a new review for a piece of music" do
+  before do
+    piece = FactoryGirl.create(:piece)
+    visit piece_path(piece)
+  end
+
+  scenario "user adds new review of music successfully" do
+    add_review
```

Comment:
It's better not to use this helper, I had to look around to multiple files to understand what your test was doing. Put in all the explicit steps of adding a new review here.